independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenArtifactResolver.java

```diff
@@ -145,6 +146,7 @@ private MavenArtifactResolver(Builder builder) throws AppModelResolverException
         if (builder.offline != null) {
             newSession.setOffline(builder.offline);
         }
+        newSession.setTransferListener(new ConsoleMavenTransferListener(System.out, true));
```

Comment:
`newSession` here might be wrapping one that is initialized by the actual Maven process, in which case it most probably already has the transfer listener in place. Perhaps it'd be better to check whether it's already present and if not initialize it that way.